app/Console/Commands/WaitingLists/SendAtcTopTenNotification.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace App\Console\Commands\WaitingLists;
+
+use App\Console\Commands\Command;
+use App\Models\Training\WaitingList;
+use App\Notifications\Training\WaitingListAtcTopTen;
+use Carbon\Carbon;
+use Illuminate\Support\Collection;
+
+class SendAtcTopTenNotification extends Command
+{
+    /**
+     * The console command name.
+     *
+     * @var string
+     */
+    protected $name = 'waitinglists:sendatctoptennotification';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Sends the top 10 members of an ATC waiting list a notification';
+
+    /**
+     * Executes all necessary console commands.
+     */
+    public function handle()
+    {
+        WaitingList::all()->filter(function ($waitingList) {
+            return $waitingList->isAtcList();
+        })->mapWithKeys(function (WaitingList $waitingList) {
```

Comment:
This seems a little inefficient (we don't have many waiting lists, but hey, best practice 😉)
It would be better to implement the filter at the database level like so (you could possibly make this a scope filter - see https://laravel.com/docs/9.x/eloquent#local-scopes, but I will leave this up to you)
```suggestion
        WaitingList::where('department', WaitingList::ATC_DEPARTMENT)->get()->mapWithKeys(function (WaitingList $waitingList) {
```

Reply:
Hi @atoff 

Thank you for the review / feedback, much appreciated!

I've updated this to use a new ATC scope, which I believe makes the most sense in this context (and may prove to be useful in the future).

I completely agree that it's best to form good habits and scope the retrieval of data at the database end, for the best performance.